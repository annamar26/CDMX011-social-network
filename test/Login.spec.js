/**
 * @jest-environment jsdom
 */
import './globals/firebase';
import Login from '../src/components/Login.js';
import { render } from '../src/utils.js';

describe('Login', () => {
  document.body.innerHTML = '<div id="root"></div>';
  const rootDiv = document.getElementById('root');
  it('should render Login page', () => {
    render(rootDiv, Login);
    expect(rootDiv.contains).toMatchSnapshot();
  });
});
