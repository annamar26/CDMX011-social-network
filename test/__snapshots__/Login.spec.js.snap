// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render Login page 1`] = `[Function]`;
